### ISSUE TITLE
The Vue language server crashed

### ISSUE THREAD

**Issue by ameesters** [octref/vetur#251]
- Platform: Linux (ubuntu 17.04)
- Vetur version: 0.8.1
- VSCode version: 1.13.1 (Shell: 1.6.6, Renderer: 56.0.2924.87, Node: 7.4.0)

**What did you expect to happen?**
When i open my vue.js project (client-side only) to start working without error.

**What did happen?**
I got a notification from VSCode:

> The Vue Language Server server crashed 5 times in the last 3 minutes. The server will not be restarted.

Were there any other errors?
Yes, this stacktrace in the console:
```
module.js:472
    throw err;
    ^

Error: Cannot find module 'lodash'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/alexander/.vscode/extensions/octref.vetur-0.8.1/client/node_modules/vue-language-server/dist/htmlServerMain.js:8:9)
    at Object.<anonymous> (/home/alexander/.vscode/extensions/octref.vetur-0.8.1/client/node_modules/vue-language-server/dist/htmlServerMain.js:224:3)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
[Info  - 9:45:49 AM] Connection to server got closed. Server will restart.
module.js:472
    throw err;
    ^

Error: Cannot find module 'lodash'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/alexander/.vscode/extensions/octref.vetur-0.8.1/client/node_modules/vue-language-server/dist/htmlServerMain.js:8:9)
    at Object.<anonymous> (/home/alexander/.vscode/extensions/octref.vetur-0.8.1/client/node_modules/vue-language-server/dist/htmlServerMain.js:224:3)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
[Info  - 9:45:49 AM] Connection to server got closed. Server will restart.
module.js:472
    throw err;
    ^

Error: Cannot find module 'lodash'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/alexander/.vscode/extensions/octref.vetur-0.8.1/client/node_modules/vue-language-server/dist/htmlServerMain.js:8:9)
    at Object.<anonymous> (/home/alexander/.vscode/extensions/octref.vetur-0.8.1/client/node_modules/vue-language-server/dist/htmlServerMain.js:224:3)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
[Info  - 9:45:49 AM] Connection to server got closed. Server will restart.
module.js:472
    throw err;
    ^

Error: Cannot find module 'lodash'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/alexander/.vscode/extensions/octref.vetur-0.8.1/client/node_modules/vue-language-server/dist/htmlServerMain.js:8:9)
    at Object.<anonymous> (/home/alexander/.vscode/extensions/octref.vetur-0.8.1/client/node_modules/vue-language-server/dist/htmlServerMain.js:224:3)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
[Info  - 9:45:50 AM] Connection to server got closed. Server will restart.
module.js:472
    throw err;
    ^

Error: Cannot find module 'lodash'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/alexander/.vscode/extensions/octref.vetur-0.8.1/client/node_modules/vue-language-server/dist/htmlServerMain.js:8:9)
    at Object.<anonymous> (/home/alexander/.vscode/extensions/octref.vetur-0.8.1/client/node_modules/vue-language-server/dist/htmlServerMain.js:224:3)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
[Error - 9:45:50 AM] Connection to server got closed. Server will not be restarted.
```

**How much does this bug affect your work?**
Medium level, i can continue to work, only without all the goodies.

Thank you for this amazing plugin, if any question or further information is needed i will provide it ASAP!

**Comment by HerringtonDarkholme** [comment]
I cannot reproduce this. I guess there is network error when you install the extension.
My suggestion is to reinstall vetur.

**Comment by octref** [comment]
Probably due to my wrongly published 0.8.1 and wouldn't happen in 0.8.2.

@ameesters Feel free to reopen if this happens to 0.8.2 too.
